plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/Plugin.java

```diff
@@ -154,4 +155,13 @@ public String toString() {
         }
         return name + " " + version + " " + url;
     }
+
+    @SuppressFBWarnings("EQ_COMPARETO_USE_OBJECT_EQUALS")
```

Comment:
please no see my comment in the main thread

Reply:
Ok, good to know. I thought in particular generating the hashCode method was going to be really complicated, but that was fairly easy.